src/vivarium/framework/randomness/manager.py

```diff
@@ -152,7 +171,7 @@ def get_seed(self, decision_point: str) -> int:
             A seed for a random number generation that is linked to Vivarium's
             common random number framework.
         """
-        return get_hash("_".join([decision_point, str(self._clock()), str(self._seed)]))
+        return get_hash("_".join([decision_point, str(self.clock()), str(self._seed)]))
```

Comment:
This creates some problems because of name mangling
https://docs.python.org/3/reference/expressions.html#private-name-mangling
How about _clock_ for the original attribute?